chore(tuto): content review (#3386)
scaleway/docs-content: tutorials/jitsi-docker/index.mdx

- An [Instance](/compute/instances/how-to/create-an-instance) running on Ubuntu Bionic Beaver (or later) with **at least 4 GB of RAM**.
- A [domain or subdomain](/network/domains-and-dns/quickstart/) pointed to your Instance

> ## Setting up the Docker

Reviewer: ```suggestion
## Setting up Docker
```

1. [Connect to your Instance](/compute/instances/how-to/connect-to-instance/) via SSH.
2. Update the package cache and upgrade the software already installed on the Instance using the `apt` package manager:
    ```
    apt update && apt upgrade -y
    ```
3. Add Docker's official GPG key to your system:
    ```
    apt-get update
    apt-get install ca-certificates curl
    install -m 0755 -d /etc/apt/keyrings
    curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
    chmod a+r /etc/apt/keyrings/docker.asc
    ```
4. Add the Docker repository to your Apt sources:
    ```
    echo \
      "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu \
      $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
      tee /etc/apt/sources.list.d/docker.list > /dev/null
    apt-get update
    ```
5. Install Docker using Apt:
    ```
    apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin docker-compose unzip
    ```

## Installing Jisti Meet

> 1. Download and extract the latest release of Jitsi Meet

Reviewer: ```suggestion
1. Download and extract the latest release of Jitsi Meet:
```

  ```
  wget $(curl -s https://api.github.com/repos/jitsi/docker-jitsi-meet/releases/latest | grep 'zip' | cut -d\" -f4)
  ```
2. Unzip the downloaded package:
  ```
  unzip <filename>
  ```
3. Enter the Jitsi Meet directory:
  ```
  cd jitsi-docker-jitsi-meet-*
  ```
4. Create a `.env` file by copying and adjusting `env.example`:
  ```
  cp env.example .env
  ```
4. Generate and configure strong passwords in the security section options of .env file by running the following bash script
  ```
  ./gen-passwords.sh
  ```

> 5. Create the required config directories for Jitsi Meet: 

Reviewer: ```suggestion
6. Create the required configuration directories for Jitsi Meet: 
```

  ```
  mkdir -p ~/.jitsi-meet-cfg/{web,transcripts,prosody/config,prosody/prosody-plugins-custom,jicofo,jvb,jigasi,jibri}
  ```

> 6. Run `docker compose up -d`, open a web browser and access the Jitsi Meet UI at `https://<YOUR_INSTANCE_IP>:8443`

Reviewer: ```suggestion
7. Run `docker compose up -d`, open a web browser and access the Jitsi Meet UI at `https://<YOUR_INSTANCE_IP>:8443`
```
